index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="bar">
+        <img src="/Images/app_bar.png"/>
+        <img src="/Images/Light.png"/ class="top";>
+    </header>
+    <div class="container">
+        <section class="head">
+            <h2>
+                Hey there. I’m Santiago. 
+                <section class="title">
+                    I’m a software developer
+                </section>   
```

Comment:
it is using the HX tag, its simply put as a separate section so it can be stylized. If I were to write something like
`<h2 class="title"> I’m a software developer </h2>` it would lose the CSS styling. Can I use div maybe?